mslib/msui/mscolab.py

```diff
@@ -1271,6 +1271,7 @@ def handle_view_close(self, value):
         for index, window in enumerate(self.active_windows):
             if window._id == value:
                 del self.active_windows[index]
+                self.raise_()
```

Comment:
I am not sure why, but this line does nothing on my end. Same for self.show(), self.activateWindow() and self.showNormal().
Maybe that is a Linux issue.
On Windows I can not even change the order of the windows. MSColab is always in front of Main and the views are always in front of MSColab.
What OS are you using for the main window to pop up after closing a window?

Reply:
I'm using a mac for development. It's usually the thing with Macs, the first created window would raise to the front.

Reply:
Unfortunately I can't test this on mac, but I will take your word for it working the way it should.